awpy/data/map_data.py

```diff
@@ -8,119 +8,110 @@
         "scale": 2.539062,
         "rotate": 1,
         "zoom": 1.3,
-        "selections": [
-            {"name": "default", "altitude_max": 10000, "altitude_min": -5},
-            {"name": "lower", "altitude_max": -5, "altitude_min": -10000},
-        ],
+        "lower_level_max": -5.0,
```

Comment:
I like this simplification, but I may offer a rewording -- perhaps `lower_level_max_units` or something. Just to put the "units" into the name (which unfortunately in this case is just generic "units").

Reply:
Sounds good!

Sidebar: I am a bit new to GitHub & commits; is it good etiquette for me to implement your feedback, or to let you implement it?

Reply:
I think it depends from person-to-person, repo-to-repo, but _usually_ the one who opened the PR will make the changes, unless it's a super small change.

Reply:
Sounds good. Will implement everything you've mentioned; they're pretty small changes and everything is still relatively fresh in my mind :D